exareme2/__init__.py

```diff
@@ -148,3 +148,17 @@ def find_flower_algorithm_folder_paths(algorithm_folders):
 flower_algorithm_folder_paths = find_flower_algorithm_folder_paths(
     FLOWER_ALGORITHM_FOLDERS
 )
+
+
+EXAFLOW_ALGORITHM_FOLDERS_ENV_VARIABLE = "EXAFLOW_ALGORITHM_FOLDERS"
+EXAFLOW_ALGORITHM_FOLDERS = "./exareme2/algorithms/exaflow"
+if exaflow_algorithm_folders := os.getenv(EXAFLOW_ALGORITHM_FOLDERS_ENV_VARIABLE):
+    EXAFLOW_ALGORITHM_FOLDERS = exaflow_algorithm_folders
```

Comment:
Do we need to have a fallback? We don't have one for exareme2.

https://www.w3schools.com/python/ref_os_getenv.asp
You can also use this one for assigning default value.

Reply:
This was an example on how exareme2 had it at line 34.